utils/renderer/electron.js

```diff
@@ -11,23 +11,37 @@ const notify = (title, body) => {
   }
 }
 
-const openFolderDialog = defaultPath => {
+const showOpenDialog = (
+  openDirectory = false,
```

Comment:
nit: this interface is a bit confusing. I was scratching my head wondering why you would want to disable opening a directory dialog, but after going through the code, I see the other side of the coin is `openFile`. I get that the appeal of this implementation is sticking with a boolean, but it requires you to remember which one is the default setting: file or directory. Suggestion: `pathType` which accepts a string: either 'file' or 'directory'. (You can still set 'file' to be the default.)

Reply:
yeah for sure makes sense, i'll update to pathType